<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor</class>
 <widget class="QMainWindow" name="monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="cpci_tree">
      <property name="headerHidden">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IO (KB)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Errors</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="control_tabs">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="capture_tab">
       <attribute name="title">
        <string>Capture</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Output file:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="runButton">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>340</y>
          <width>110</width>
          <height>33</height>
         </rect>
        </property>
        <property name="text">
         <string>Run</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="outfile_lable">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>10</y>
          <width>251</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>none</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="status_tab">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>81</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Bad Events:</string>
        </property>
       </widget>
       <widget class="QLabel" name="bad_events_lable">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>121</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Bad Event Ratio:</string>
        </property>
       </widget>
       <widget class="QLabel" name="bad_events_ratio_lable">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>50</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0%</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Bus Errors:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Aurora Errors:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>101</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>DMA Errors:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>141</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Watchdog Overflow:</string>
        </property>
       </widget>
       <widget class="QLabel" name="bus_errors_lable">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="aurora_errors_lable">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>110</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="dma_errors_lable">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="watchdog_errors_lable">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>170</y>
          <width>171</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="reset_stats_button">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>350</y>
          <width>96</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="misc_tab">
       <attribute name="title">
        <string>Options</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolBox" name="option_toolbox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>347</width>
             <height>275</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Storage</string>
           </attribute>
           <widget class="QCheckBox" name="flush_to_disk">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>131</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Flush to disk</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="store_meta">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>131</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Store metadata</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>347</width>
             <height>275</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Link</string>
           </attribute>
           <widget class="QCheckBox" name="rst_one_error">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>10</y>
              <width>131</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Reset on error</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="flowctl_box">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>111</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Flow control</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QCheckBox" name="vetoctl_box">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>50</y>
              <width>111</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>Veto Clear</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="AboutPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>347</width>
             <height>275</height>
            </rect>
           </property>
           <attribute name="label">
            <string>About</string>
           </attribute>
           <widget class="QLabel" name="contact_lable">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>250</y>
              <width>141</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>&lt;a href=&quot;mailto:sin8@hawaii.edu&quot;&gt;Contact Developer&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>181</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>Driver Version:</string>
            </property>
           </widget>
           <widget class="QLabel" name="driver_version_lable">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>0</y>
              <width>78</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>1.0</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>120</x>
              <y>140</y>
              <width>101</width>
              <height>111</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>:/img/icon.png</pixmap>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSet_Output"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConsole">
    <property name="title">
     <string>Console</string>
    </property>
    <addaction name="actionShow_console"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuConsole"/>
  </widget>
  <action name="actionSet_Output">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShow_console">
   <property name="text">
    <string>Toggle console</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
